README.md

```diff
@@ -181,6 +181,23 @@ Class names like `Options` and `Level` might be too common, if you get a conflic
     })
     ...
 
+#### Angular Universal
+
+To use it with Angular Universal add angular2-logger to App.module normaly and in node.module provide these custom options.   
+
+```javascript
+import { Options } from "angular2-logger/core";
+
+@NgModule({
+  bootstrap: [ AppComponent ],
+  imports: [ AppModule, ... ],
+  providers: [
+    { provide: Options, useValue: { global: false, store: false } }, // stop angular2-logger from defining global variables.
```

Comment:
@langley-agm  , yes, I understand that. I don't know if you are familiar with [Angular Universal](https://github.com/angular/universal) ? When using Angular Universal your code will be rendered both in the client and in the server. When the code is rendering in the server it should not try to use the ‘Window’ object because it will not be available there.
So the idea is that it should be possible to configure angular2-logger in such a way that it will never try to use the Window API, i.e. { global: false, store: false }.
The problem is that even though “global” and “store” are false angular2-logger will still try to access the Window object because of the call to _loadLevel().
My change simply remove that call so that the code window.localStorage is never executed if “store” is false.
With this it is possible to have two angular2-logger configuration, one for the Server and one for the Client. The one for the server has to have “global” and “state” set to false and the one for the client can be in whatever way the user wants.